### PR TITLE
Add new `Cards` component

### PR DESCRIPTION
**Pull request by elliotmjackson**
Adds a new component called Cards, built off an existing docusaurus component but a little less prescriptive, allowing a caller to supply the name, url and description to build the card.

image

These cards are created in mdx using the following...

```js
import { Card, Cards } from "@site/src/components/Cards";

<Cards>
  <Card
    name="🖌 Tutorial"
    url="/tour/introduction"
    description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
  />
  <Card
    name="🏗 How To"
    url="/bsr/overview"
    description="Nisl tincidunt eget nullam non. Sed cras ornare arcu dui vivamus. Id neque aliquam vestibulum morbi blandit. Turpis nunc eget lorem dolor sed."
  />
</Cards>
```

### CODE REVIEW

File: src/components/Cards/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import clsx from 'clsx';
+import Link from '@docusaurus/Link';
+
+interface Props {
+  name: string;
+  url: string;
+  description: string;
+}
+
+function Card({name, url, description}: Props) {
```
Comment (paul-sachs) [inline src/components/Cards/index.tsx:11]: I'd suggest exporting this guy as well, and change Cards to be even dumber:

```tsx
export const Cards: FC<PropsWithChildren> = ({ children }) => {
  return <div className="row">{children}</div>;
}
```

And then the usage is more like:


```tsx
<Cards>
  <Card name="..." url="..." description="..." />
  ...
</Cards>
```

Comment (elliotmjackson) [reply src/components/Cards/index.tsx:11]: ooo much better

Comment (elliotmjackson) [reply src/components/Cards/index.tsx]: resolved in [e7127a9](https://github.com/bufbuild/docs.buf.build/pull/340/commits/e7127a9850264de0e3408497fdb98d857a9cee73)

**Comment by paul-sachs** [comment]
@elliotmjackson You can also revert your package-lock.json change. since you've removed the package.json change.

**Review by paul-sachs — APPROVED** [review]

**Pull request merged**